web/modules/custom/joinup_core/joinup_core.post_update.php

```diff
@@ -111,9 +111,4 @@ function joinup_core_post_update_update_cached_timers() {
     $editable->setData($data);
     $editable->save();
   }
-
-  // Refresh the snapshot properly.
-  $config_sync_snapshotter = \Drupal::service('config_sync.snapshotter');
-  $config_sync_snapshotter->deleteSnapshot();
-  $config_sync_snapshotter->refreshSnapshot();
```

Comment:
I would still remove the entire post update function, and especially the part that sets the config. The code to refresh the snapshot could stay, but even that seems unnecessary. This problem will sort itself out soon enough, if not on the first deployment, then on the second.